## Nonlinear Model Predictive Control Params
controller/NMPC: 
# Optimization Setup
  dt: 0.1 # Sampling time [s]
  N:  15  # Prediction Horizon

  # Weight Matrices
  Q_x:      3.0 # X cost
  Q_y:      3.0 # Y cost
  Q_theta:  1.0 # Theta cost

  R_v:      0.01 # Linear speed cost
  R_omega:  0.5 # Angular speed cost

  # Boundary Action Control
  v_max:      0.8 # [m/s]
  omega_max:  1.2 # [rad/s]

## Odometry Params
odom:
  topic: "/terrasentia/will/odom"
  frame_id: "/terrasentia/will/odom"
